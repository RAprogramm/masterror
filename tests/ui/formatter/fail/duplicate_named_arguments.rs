use masterror::Error;

#[derive(Debug, Error)]
#[error("{value}", value = self.value, value = self.value)]
struct DuplicateNamedArgumentError {
    value: &'static str,
}

fn main() {}
