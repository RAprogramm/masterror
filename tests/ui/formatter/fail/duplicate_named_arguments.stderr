error: duplicate format argument `value`
 --> tests/ui/formatter/fail/duplicate_named_arguments.rs:8:40
  |
8 | #[error("{value}", value = self.value, value = self.value)]
  |                                        ^^^^^
